docs/proposals/proposal-trigger-and-deploy.md

```diff
@@ -0,0 +1,286 @@
+# Proposal-001 - Trigger and Deploy GitHub Action workflow from an upstream CNCF project
+
+To trigger our benchmarking task to run when a particular CNCF project gets certain kinds of event, such as a new `release`.
+
+- Tracking issue: [#83](https://github.com/cncf-tags/green-reviews-tooling/issues/83)
+- Implementation issue: [#84](https://github.com/cncf-tags/green-reviews-tooling/issues/84)
+
+## Authors
+
+- @rossf7
+- @dipankardas011
+
+## Status
+
+implementable
+
+<!--
+Must be one of provisional, implementable, implemented, deferred,
+rejected, withdrawn, or replaced.
+-->
+
+## Table of Contents
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+  - [Linked Docs](#linked-docs)
+- [Proposal](#proposal)
+  - [User Stories](#user-stories)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Graduation Criteria (Optional)](#graduation-criteria-optional)
+- [Drawbacks (Optional)](#drawbacks-optional)
+- [Alternatives](#alternatives)
+- [Infrastructure Needed](#infrastructure-needed)
+<!-- /toc -->
+
+## Summary
+
+This proposal focuses on automating the Green Reviews pipeline for Falco by 
+defining a trigger mechanism, involving the Falco team in the implementation, 
+deploying Falco using Flux, and testing the deployment process. In future the 
+pipeline will support more CNCF projects as they are onboarded.
+
+The proposal also includes considerations for a phased implementation of the 
+automation, starting with manual triggering followed by automation via a webhook.
+
+The high level architecture is shown in this diagram.
+
+![wg green reviews workflow vision](./files/green-reviews-wg-workflow-vision.webp)
```

Comment:
@dipankardas011 I added the diagram as is from the blog post. I think it adds context and my bad for not including any diagrams in the previous version! 

I think an updated diagram focused on trigger & deploy would still be useful if you have time. WDYT?

Reply:
Sure